src/Client/Web/Bit.Client.Web.BlazorUI/Components/ColorPicker/AlphaType.cs

```diff
@@ -1,10 +1,20 @@
-﻿
-namespace Bit.Client.Web.BlazorUI
+﻿namespace Bit.Client.Web.BlazorUI
 {
     public enum AlphaType
```

Comment:
please ignore all types related to BitColorPicker.
this component is under heavy refactoring :)